charts/cloudflare-tunnel/templates/deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - tunnel
+            - '--protocol'
+            - {{ .Values.cloudflare.protocol }}
```

Comment:
Would it be better that this is moved in the config.yaml which is sourced via the configmap?